docs/how-to-guides/themes/theme-json.md

```diff
@@ -1084,6 +1085,41 @@ Pseudo selectors `:hover`, `:focus`, `:visited`, `:active`, `:link`, `:any-link`
 	}
 ```
 
+#### Variations
+
+A block can have a "style variation", as defined per the [block.json specification](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/#styles-optional). Theme authors can define the style attributes for an existing style variation using the `theme.json`. Styles for unregistered style variations will be ignored.
```

Comment:
Not a blocker, but a tiny grammatical nit: should it be `using the theme.json file` or simply `using theme.json`?

Reply:
updated at https://github.com/WordPress/gutenberg/pull/49826/commits/9c4cffee385738b290ab76a60ddcf90f502a15c0